src/app/archive/patch_migrated_db.sql

```diff
@@ -2,5 +2,5 @@
 -- in this PR https://github.com/MinaProtocol/mina/pull/14407 
 ALTER TABLE zkapp_permissions RENAME COLUMN set_verification_key TO set_verification_key_auth;
 ALTER TABLE zkapp_permissions ADD COLUMN set_verification_key_txn_version int;
-UPDATE      zkapp_permissions SET set_verification_key_txn_version = 1;
-ALTER TABLE zkapp_permissions ALTER COLUMN set_verification_key_txn_version SET NOT NULL;
\ No newline at end of file
+UPDATE      zkapp_permissions SET set_verification_key_txn_version = 2;
```

Comment:
I wonder if we need this script at all since it is changing already migrated database while i don't know if we have any (or anyone has ir)

Reply:
https://github.com/MinaProtocol/mina/issues/15366